src/sentry/api/endpoints/release_thresholds/release_threshold_details.py

```diff
@@ -11,14 +12,34 @@
 from sentry.api.exceptions import ResourceDoesNotExist
 from sentry.api.serializers import serialize
 from sentry.models.project import Project
+from sentry.models.release_threshold.constants import (
+    THRESHOLD_TYPE_STR_TO_INT,
+    TRIGGER_TYPE_STRING_TO_INT,
+    ReleaseThresholdType,
+)
+from sentry.models.release_threshold.constants import TriggerType as ReleaseThresholdTriggerType
 from sentry.models.release_threshold.release_threshold import ReleaseThreshold
 
 
+class ReleaseThresholdPUTSerializer(serializers.Serializer):
+    threshold_type = serializers.ChoiceField(choices=ReleaseThresholdType.as_str_choices())
+    trigger_type = serializers.ChoiceField(choices=ReleaseThresholdTriggerType.as_str_choices())
+    value = serializers.IntegerField()
+    window_in_seconds = serializers.IntegerField()
+
+    def validate_threshold_type(self, threshold_type: str):
+        return THRESHOLD_TYPE_STR_TO_INT[threshold_type]
```

Comment:
This is pretty cool, I didn't know you could validate and convert the input to the correct serializer field type